src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -711,9 +711,9 @@ private IEnumerable<ManifestItemWithContext> ExportManifest(HostService hostServ
                         {
                             Logger.LogDiagnostic($"Processor {hostService.Processor.Name}: Saving...");
                             m.BaseDir = context.BuildOutputFolder;
-                            if (m.PathRewriter != null)
+                            if (m.FileAndType.SourceDir != m.FileAndType.DestinationDir)
                             {
-                                m.File = m.PathRewriter(m.File);
+                                m.File = (RelativePath)m.FileAndType.DestinationDir + (((RelativePath)m.File) - (RelativePath)m.FileAndType.SourceDir);
```

Comment:
remove `Move` in `RelativePath`, since parameters don't make sense.

---
In reply to: [77533996](https://github.com/dotnet/docfx/pull/669#discussion_r77533996) [](ancestors = 77533996)